test/expect-tests/dune_file_watcher/dune_file_watcher_tests_patterns.ml

```diff
@@ -2,7 +2,9 @@ let printf = Printf.printf
 
 let test string =
   printf "should_exclude(%s) = %s\n" string
-    (Dune_file_watcher.For_tests.should_exclude string |> Bool.to_string)
+    (Dune_file_watcher.For_tests.should_exclude string
+       Dune_config.standard_watch_exclusions
+    |> Bool.to_string)
```

Comment:
You can replace the second `%s` with `%b`.

Reply:
done